libs/blocks/chart/chart.js

```diff
@@ -33,9 +33,10 @@ const parseValue = (value) => (Number.isInteger(Number(value)) ? parseInt(value,
 
 export function processDataset(data) {
   const dataset = {};
-
-  // Remove group and unit from headers
-  const headers = Object.keys(data[0]).filter((header) => !['unit', 'group', 'color'].includes(header.toLowerCase()));
+  const optionalHeaders = ['unit', 'group', 'color'];
+  const headers = Object.keys(data[0]).filter((header) => (
+    !optionalHeaders.includes(header.toLowerCase())
+  ));
```

Comment:
Since headers are used in a few places do you think it'd be better to process the headers in `fetchData` so you also have `data.headers` and `data.optionalHeaders`.  The optionalHeaders could be converted to all lowercase keys.  I'm not that familiar with the code so might not be a good idea.

Reply:
I think that's an idea worth looking into. It would increase the scope of this ticket because data is used in so many places, so I'll create an optimization ticket to look into it